--- linux-2.6.26/net/wireless/wext.c.orig	2008-07-13 17:51:29.000000000 -0400
+++ linux-2.6.26/net/wireless/wext.c	2021-03-27 09:00:43.926792944 -0400
@@ -1224,11 +1224,11 @@
 	if (descr->header_type == IW_HEADER_TYPE_POINT) {
 		/* Check if number of token fits within bounds */
 		if (wrqu->data.length > descr->max_tokens) {
-			printk(KERN_ERR "%s (WE) : Wireless Event too big (%d)\n", dev->name, wrqu->data.length);
+			printk(KERN_ERR "%s (WE) : Wireless Event (cmd=0x%04X) too big (%d)\n", dev->name, cmd, wrqu->data.length);
 			return;
 		}
 		if (wrqu->data.length < descr->min_tokens) {
-			printk(KERN_ERR "%s (WE) : Wireless Event too small (%d)\n", dev->name, wrqu->data.length);
+			printk(KERN_ERR "%s (WE) : Wireless Event (cmd=0x%04X) too small (%d)\n", dev->name, cmd, wrqu->data.length);
 			return;
 		}
 		/* Calculate extra_len - extra is NULL for restricted events */
